api/nodes/serializers.py

```diff
@@ -63,10 +40,12 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
-    class Meta:
-        type_ = 'nodes'
+    registration = ser.BooleanField(source='is_registration', read_only=True)
```

Comment:
The first three I checked, Stripe, Twilio, and Github, all neglect to use the `is_` prefix.